### PR TITLE
[ALLUXIO-1922] collapsed identical catches to multi-catch

### PR DESCRIPTION
**Pull request by sandeep-n**
https://alluxio.atlassian.net/browse/ALLUXIO-1922

collapsed identical catches to multi-catch

### CODE REVIEW

File: core/server/src/main/java/alluxio/worker/block/BlockWorker.java [hunk 1]
```diff
@@ -14,12 +14,7 @@
 import alluxio.Configuration;
 import alluxio.Constants;
 import alluxio.Sessions;
-import alluxio.exception.AlluxioException;
-import alluxio.exception.BlockAlreadyExistsException;
-import alluxio.exception.BlockDoesNotExistException;
-import alluxio.exception.ConnectionFailedException;
-import alluxio.exception.InvalidWorkerStateException;
-import alluxio.exception.WorkerOutOfSpaceException;
+import alluxio.exception.*;
```
Comment (ifcharming) [inline core/server/src/main/java/alluxio/worker/block/BlockWorker.java]: Please follow the import orders in Coding Style [here](http://alluxio.org/documentation/master/en/Contributing-to-Alluxio.html)

- For Eclipse to organize your imports correctly, configure “Organize Imports” to look like [this](http://alluxio.org/documentation/master/resources/eclipse_imports.png)
- For IntellJ, configure in Preferences->Code Style->Java->Imports->Import Layout according to this [order](http://alluxio.org/documentation/master/resources/intellij_imports.png)

File: core/server/src/main/java/alluxio/worker/block/BlockWorker.java [hunk 2]
```diff
@@ -290,10 +283,8 @@ public void commitBlock(long sessionId, long blockId)
       Long bytesUsedOnTier = storeMeta.getUsedBytesOnTiers().get(loc.tierAlias());
       mBlockMasterClient.commitBlock(WorkerIdRegistry.getWorkerId(), bytesUsedOnTier,
           loc.tierAlias(), blockId, length);
-    } catch (IOException ioe) {
+    } catch (IOException | ConnectionFailedException ioe) {
```
Comment (ifcharming) [inline core/server/src/main/java/alluxio/worker/block/BlockWorker.java]: nit: s/ioe/e ?

**Comment by ifcharming** [comment]
Thank you @sandeep-n for the PR! I left a few minor comments.

Per Alluxio convention, can you please rename the PR title to be "[ALLUXIO-1922] collapsed identical catches to multi-catch". It would be also great if you could attach the JIRA ticket link in the PR description: https://alluxio.atlassian.net/browse/ALLUXIO-1922  

Thanks!

**Comment by ifcharming** [comment]
ok to test

**Comment by ifcharming** [comment]
Reverting the import changes should fix the build failure :)

File: core/server/src/main/java/alluxio/worker/block/BlockWorker.java (hunk 1 above)
Comment (jsimsa) [inline core/server/src/main/java/alluxio/worker/block/BlockWorker.java]: we do not use wildcard imports in Alluxio ... please list the imported exceptions explicitly

File: core/server/src/main/java/alluxio/worker/block/BlockWorker.java [hunk 3]
```diff
@@ -37,21 +32,19 @@
 import alluxio.worker.block.meta.BlockMeta;
 import alluxio.worker.block.meta.TempBlockMeta;
 import alluxio.worker.file.FileSystemMasterClient;
-
 import com.google.common.base.Throwables;
 import org.apache.thrift.TProcessor;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.annotation.concurrent.NotThreadSafe;
```
Comment (jsimsa) [inline core/server/src/main/java/alluxio/worker/block/BlockWorker.java]: undo this change

**Comment by jsimsa** [comment]
@sandeep-n thank you for the PR, left some comments

**Comment by sandeep-n** [comment]
@ifcharming @jsimsa Thanks for the feedback!

**Comment by jsimsa** [comment]
LGTM
